apps/emqx_postgresql/src/emqx_postgresql.erl

```diff
@@ -679,6 +709,25 @@ prepare_sql_to_conn(Conn, [{Key, {SQL, _RowTemplate}} | Rest], Statements) when
             {error, export_error(TranslatedError)}
     end.
 
+-ifdef(TEST).
+test_maybe_inject_prepared_statement_already_exists(Conn, Key, SQL) ->
```

Comment:
Nit: we might emulate this as well by using `meck` or even registering the action name before creating the action?